cli/azd/cmd/provision.go

```diff
@@ -40,6 +41,12 @@ func (i *provisionFlags) bindNonCommon(local *pflag.FlagSet, global *internal.Gl
 
 func (i *provisionFlags) bindCommon(local *pflag.FlagSet, global *internal.GlobalCommandOptions) {
 	local.BoolVar(&i.preview, "preview", false, "Preview changes to Azure resources.")
+	local.BoolVar(
+		&i.ignoreDeploymentState,
+		"no-state",
```

Comment:
In @savannahostrowski's draft blog post for this feature, we said this was called `--no-cache`.  I slightly prefer that to `--no-state`.  But if there was some discussion I missed where we landed on `--no-state` that's fine, we'll just need to go update the blog post.